### DISCUSSION TITLE
week 6 - Chess game

### DISCUSSION THREAD

**Discussion by DavidDHBW** [DavidDHBW/codespaces-jupyter#8] [General]
Chess game:
Aufteilung in 2 skripten für mehr Übersicht. Ein Skript um inptut/output zu verwerten und ein anderes um das Brett zu speichern, wer am zug ist, movelog usw.

ziele für die woche wären eine vollständigere engine. Grafiken für das board und figuren zu implementieren.

Update: 
-Habe open source datei mit schachfiguren gefunden und ins Projekt geladen.
-Habe eine loading function für die png images geschrieben.

Probleme:
finde nicht heraus wie man ein python package erstellt in der browser version von vs

[code](https://github.com/DavidDHBW/codespaces-jupyter/tree/main/Python/Chess)

**Comment by Domidhbw** [comment]
Uhh  Chess als projekt klingt mega cool. Kennst du dieses video?
https://www.youtube.com/watch?v=_vqlIPDR2TU
